src/unix/linux_like/linux/gnu/mod.rs

```diff
@@ -453,6 +453,43 @@ s! {
         pub aio_flags: ::__u32,
         pub aio_resfd: ::__u32,
     }
+
+    // netinet/tcp.h
+
+    pub struct tcp_info {
+        pub tcpi_state: u8,
+        pub tcpi_ca_state: u8,
+        pub tcpi_retransmits: u8,
+        pub tcpi_probes: u8,
+        pub tcpi_backoff: u8,
+        pub tcpi_options: u8,
+        // bitfields 4: tcp_snd_wscale, tcp_rcv_wscale
+        pub tcpi_snd_rcv_wscale: u8,
```

Comment:
```suggestion
        /// This contains the bitfields `tcpi_snd_wscale` and `tcpi_rcv_wscale`. Each is four bits.
        pub tcpi_snd_rcv_wscale: u8,
```

Let's make this show up in documentation